core/sdk-core/src/main/java/software/amazon/awssdk/core/batchmanager/BatchOverrideConfiguration.java

```diff
@@ -116,6 +144,28 @@ public Builder maxBatchItems(Integer maxBatchItems) {
             return this;
         }
 
+        /**
+         * Define the maximum number of batchKeys to keep track of.
```

Comment:
Can we add more javadocs to clarify the definition of batchKey?

Reply:
Added javaodcs to clarify definition of batchKey. Also added example of how batchKey might be calculated with SQS.